### DISCUSSION TITLE
Philips Hue: Matter update available for testing now

### DISCUSSION THREAD

**Discussion by lboue** [mfucci/node-matter#129] [Matter compatible devices / controllers]
### Philips Hue: Matter update available for testing now

Blog post: [Philips Hue: Matter update available for testing now
](https://hueblog.com/2022/11/04/philips-hue-matter-update-available-for-testing-now/)

> The most important news around Philips Hue: The Hue Bridge is already certified for Matter. What is now missing is a software update for the bridge. Once that is installed and armed, you can use all the switches and lights connected to the Hue Bridge in the Matter ecosystem. There are only two exceptions: The [Philips Hue Play HDMI Sync Box](https://hueblog.com/2022/05/27/hue-sync-box-soon-integrated-into-the-regular-hue-app/) and the [Philips Hue Tap Dial Switch](https://hueblog.com/2022/06/21/hands-on-with-the-new-philips-hue-tap-dial-switch/), which are not yet supported by the current Matter version.
The said software update will be part of the regular updates for the Philips Hue Bridge and is expected to be available in the first quarter of 2023. If you can’t wait yet, a beta version is already available. You can activate it on the [Philips Hue Developer website](https://developers.meethue.com/matter/). But don’t expect things to start rolling minutes later – even though the official launch was on 3 November, it will still take a few weeks for Matter to really get rolling.

Official release in the first quarter of 2023

**Comment by mfucci** [comment]
Thanks! Please update here if you succeed to have one of those working at your house using Matter!

**Reply by lboue** [reply]
Yes it's works.
![image](https://user-images.githubusercontent.com/938089/204634234-a3258e6a-40ce-4bb2-bc6e-4536922b93b7.jpeg)

**Reply by mfucci** [reply]
Are you sure they are actually connecting using Matter? The iPhone supports many other protocols.

Try to scan for "_matter._tcp" over MDNS to see if they have been provisioned as Matter devices.
The linux command is: `avahi-browse _matter._tcp` and there should be something similar for MacOsX / Windows

If they are provisioned as Matter devices, you see something like:
+ enp0s31f6 IPv4 24C08E412A317472-0000000000000022             _matter._tcp         local
+ enp0s31f6 IPv4 24C08E412A317472-0000000000000021             _matter._tcp         local
+ lo IPv4 24C08E412A317472-0000000000000022             _matter._tcp         local
+ lo IPv4 24C08E412A317472-0000000000000021             _matter._tcp         local

**Comment by lboue** [comment]
I'm 100% sure of that because:
1) I provided the Matter pairing code during the process:
2) According to the naming of the paired accessories / Sometimes I am getting the Hue devices name, sometimes a generic name
3) According to the characteristics of the accessories

```
$ avahi-browse _matter._tcp
+  wlan0 IPv6 B03E7C9E75E106A9-3F1FD177161FA639             _matter._tcp         local
+  wlan0 IPv6 7D79D89ACC7FD7E2-0000000061AB6ED4             _matter._tcp         local
+  wlan0 IPv6 B03E7C9E75E106A9-000000008219B215             _matter._tcp         local
+  wlan0 IPv4 B03E7C9E75E106A9-3F1FD177161FA639             _matter._tcp         local
+  wlan0 IPv4 7D79D89ACC7FD7E2-0000000061AB6ED4             _matter._tcp         local
+  wlan0 IPv4 B03E7C9E75E106A9-000000008219B215             _matter._tcp         local
```

**Comment by lboue** [comment]
![IMG-0809](https://user-images.githubusercontent.com/938089/206641030-3ca6bb88-eef9-488c-bbc9-873bfc9c2372.jpg)
![IMG-0808](https://user-images.githubusercontent.com/938089/206641044-f4bf1966-2ceb-4982-86e7-df3168688945.jpg)

**Reply by mfucci** [reply]
Great, thanks!
I am close to being able to test node-matter against Google Home so more updates in a couple of days!

**Reply by mfucci** [reply]
Is this the bridge you have?
https://www.amazon.com/Philips-Hue-Dimmable-Starter-Compatible/dp/B083TP78ZT/

For $50, I can buy one and play with it...

**Reply by lboue** [reply]
Yes I own hue bridge version 2 (square)
